homeassistant/components/tado/device_tracker.py

```diff
@@ -49,27 +49,21 @@ def get_scanner(hass: HomeAssistant, config: ConfigType) -> TadoDeviceScanner |
 class TadoDeviceScanner(DeviceScanner):
     """Scanner for geofenced devices from Tado."""
 
-    def __init__(self, hass, config):
+    def __init__(self, hass: HomeAssistant, config) -> None:
         """Initialize the scanner."""
         self.hass = hass
-        self.last_results = []
+        self.last_results = list[str]
 
         self.username = config[CONF_USERNAME]
         self.password = config[CONF_PASSWORD]
+        self.fallback = CONST_OVERLAY_TADO_DEFAULT
```

Comment:
We're only using these values here, can we remove them from the `self.`?

Reply:
I suppose it can be directly put in the TadoConnector, yes. No much of a difference.

Reply:
It has some performance improvements (and code is cleaner)